### ISSUE TITLE
Marker for points (as map features)

### ISSUE THREAD

**Issue by Lucas-Nan** [amosproj/amos2024ss04-building-information-enhancer#165]
## Description
Points should be able to be displayed with markers in the map

## Acceptance criteria
* Display multiple markers at specific points at different zoom levels

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
done with basic default marker - may be improved then with custom icons
